Debian/bin/lib/Cyrus/Docker/Command/shell.pm

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/perl
+use v5.36.0;
+
+use utf8;
+
+package Cyrus::Docker::Command::shell;
+use Cyrus::Docker -command;
+use Term::ANSIColor qw(colored);
+
+sub do_motd {
+  my $menu = <<~'END';
+            /////  ||||
+          /////    |||| Cyrus IMAP docker image
+        /////      |||| Run cyrus-docker (or "cyd") as:
+      /////        ||||
+    /////          ||||  • cyd checkout - check out cyrus-imapd.git
+    \\\\\          ||||  • cyd build    - build your checked out cyrus-imapd
+      \\\\\        ||||  • cyd test     - run the cyrus-imapd test suite
+        \\\\\      ||||
+          \\\\\    ||||
+            \\\\\  ||||
+  END
```

Comment:
Would be nice if this gave a hint about how to get a running shell

Reply:
I've added something to warn when -t was not passed.  You can't really detect `-i` being missing, AFAICT, but I expect it'll be rare to run -t without -i.

Reply:
Hm, isn't that "keep STDIN open", so STDIN would be closed?

Reply:
No, STDIN will be open and fd 0, but it isn't "the user".  I did a bunch of looking for a good answer to this, and found nothing.  Better ideas welcome.

Reply:
Looks like `-p *STDIN` will tell you if it's connected (-i) or not based on testing. But probably no big deal. (however, if you pass -t and no -i, the terminal is unusable. Amazing.)